### PR TITLE
Processors test

### PR DESCRIPTION
**Pull request by schalkms**
Implemented test discussed in #433

### CODE REVIEW

File: detekt-cli/src/test/kotlin/io/gitlab/arturbosch/detekt/cli/FileProcessorLocatorTest.kt [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ */
+class FileProcessorLocatorTest {
+
+	private val packageName = "io.gitlab.arturbosch.detekt.core.processors"
+
+	@Test
+	fun containsAllRuleProviders() {
+		val path = Paths.get(resource(""))
+		val locator = FileProcessorLocator(ProcessingSettings(path))
+		val providers = locator.load()
```
Comment (Mauin) [inline detekt-cli/src/test/kotlin/io/gitlab/arturbosch/detekt/cli/FileProcessorLocatorTest.kt]: Shouldn't this be `processors`?

File: detekt-cli/src/test/kotlin/io/gitlab/arturbosch/detekt/cli/FileProcessorLocatorTest.kt [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+/**
+ * This tests the existence of all metric processors in the META-INF config file in the core package
+ */
+class FileProcessorLocatorTest {
+
+	private val packageName = "io.gitlab.arturbosch.detekt.core.processors"
+
+	@Test
+	fun containsAllRuleProviders() {
```
Comment (Mauin) [inline detekt-cli/src/test/kotlin/io/gitlab/arturbosch/detekt/cli/FileProcessorLocatorTest.kt]: `containsAllProcessors`?

Comment (schalkms) [reply detekt-cli/src/test/kotlin/io/gitlab/arturbosch/detekt/cli/FileProcessorLocatorTest.kt]: Thanks for spotting!

File: detekt-cli/src/test/kotlin/io/gitlab/arturbosch/detekt/cli/FileProcessorLocatorTest.kt (hunk 1 above)
Comment (schalkms) [reply detekt-cli/src/test/kotlin/io/gitlab/arturbosch/detekt/cli/FileProcessorLocatorTest.kt]: This is debatable I think.

Comment (arturbosch) [reply detekt-cli/src/test/kotlin/io/gitlab/arturbosch/detekt/cli/FileProcessorLocatorTest.kt]: All thins the locator loads are processors, so processors makes sense :D

**Pull request merged**